src/main/java/tasks/TaskList.java

```diff
@@ -0,0 +1,75 @@
+package tasks;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> tasks = new ArrayList<Task>();
+
+    /**
+     *
+     * @param idx index of task in the TaskList
+     * @return Task object
+     */
+    public Task get(int idx) {
+        return tasks.get(idx);
+    }
+
+    /**
+     * Adds task into arraylist
+     *
+     * @param task Task object
+     */
+    public void add(Task task) {
+        tasks.add(task);
+    }
+
+    /**
+     * Removes task from arraylist
+     * @param idx index of task in arraylist to be removed
+     */
+    public void delete(int idx) {
+        tasks.remove(idx);
+    }
+
+    /**
+     * Marks task in arraylist
+     * @param idx index of task in arraylist to be marked
+     */
+    public void mark(int idx) {
+        tasks.get(idx).mark();
+    }
+
+    /**
+     * Unmarks task in arraylist
+     * @param idx index of task in arraylist to be unmarked
+     */
+    public void unmark(int idx) {
+        tasks.get(idx).unmark();
+    }
+
+    /**
+     *
+     * @param idx index of task in arraylist
+     * @return String representation of task
+     */
+    public String taskDesc(int idx) {
```

Comment:
Maybe you could change the function name to a verb?
```suggestion
    public String getTaskDesc(int idx) {
```